test/tests/reporters/perf_graph_reporter/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  design = 'PerfGraphReporter.md'
+  issues = '#16256'
+
+  [test]
+    type = 'RunApp'
+    input = 'perf_graph_reporter.i'
+    requirement = 'The system shall include the ability to report performance information'
```

Comment:
use a JSONDiff?

Reply:
Can't really do a diff on performance data. Not a good idea to diff on things like time. I'll add basic checks here at some point that make sure the json looks right

Reply:
umm true for the numeric field. But could look at the text output. Havent used Jsondiff though

Reply:
You can set `rel_err` to something really high. This will basically ignore numeric comparisons, but still check everything else.